docker-compose.ci.yml

```diff
@@ -7,6 +7,24 @@ services:
     volumes: !reset
     volumes:
       - artifacts:/opt/app/artifacts
+      - selenium_downloads:/home/seluser/Downloads
+
+  selenium:
+    image: selenium/standalone-chrome:latest
+    ports:
```

Comment:
It's not necessary to expose ports in the CI environment. Ports are automatically exposed to other services on the same Docker network. Mapping them in this way is only useful when trying to access ports from the host machine, which is not possible or necessary when running in CI.